### ISSUE TITLE
Moar domains

### ISSUE THREAD

**Issue by ypkang** [yqzhang/WebPage#7]
Based on the recent addition of `yunqi.cat` and un-precedent level of interests from the community in this new direction of this project, I propose looking into expanding the domain of the website to include
```
yunqi.wtf
```
Relevant information can be found [here](https://www.name.com/domains/wtf)

Running list of sample websites:
- [destiny.wtf](http://destiny.wtf/)

**Comment by jhauswald** [comment]
@ypkang I agree. Here is a more [comprehensive list](https://en.wikipedia.org/wiki/List_of_Internet_top-level_domains). A few examples:
- yunqi.kiwi
- yunqi.life
- yunqi.love
- yunqi.porn
- yunqi.rocks

@yqzhang could you work on securing these please? We can then implement the necessary websites.
